pages/index.tsx

```diff
@@ -259,17 +261,24 @@ const HomePage = ({
                     </Link>
                   </WrapItem>
                 ))}
-              <Box
-                p='4'
-                border='1px dashed'
-                borderColor={useColorModeValue('teal.200', 'teal.500')}
-                bg={useColorModeValue('teal.50', 'whiteAlpha.200')}
-                rounded='md'
-              >
-                <Box as='span' mr='1' role='img'>
-                  💖
-                </Box>{' '}
-                {t('homepage.your-company')}
+              <Box as='li'>
```

Comment:
Thanks for your feedback. Thought its easier to not use this because I need to overwrite the stlyes of WrapItem for this one element. Will have a closer look into it in a few minutes.

Reply:
Yesm you were right. Also fixed the hover bg on light mode to teal.200 instead of default button hover bg.
I force-pushed btw to keep it small. :smile: